src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImplTest.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.ons.ctp.response.collection.exercise.client.impl;
```

Comment:
There is already an ActionSvcClientImplTest.java file with other tests for that class (the name probably should be ActionSvcRestClientImplTest.java though)

Reply:
I think the older tests should probably be in here rather than in `test/.../service/impl` but that can probably be done as part of a wider cleanup of the tests/improvements to structure